web/styles/dark_theme.css

```diff
@@ -89,14 +93,21 @@
         box-shadow: inset 0 1px 0 hsl(0deg 0% 0% / 20%);
     }
 
-    .enter_sends_choices,
```

Comment:
`.enter_sends_choice` has no `kbd` element. The `send_later_popover.hbs` file was refactored in this commit e0ec522f702e5b64f2e5b0e37dbae4ce89dd53bf.